src/header/header-buttons.scss

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-header-brand {
+  height: 100%;
+  display: inline-flex;
+  align-items: center;
+  flex-shrink: 0;
+  white-space: nowrap;
+  background-color: transparent;
+  box-sizing: border-box;
+
+  @at-root {
+    a#{&}, // rendered as anchor element
+    button#{&}, // rendered as button element
+    &[role='button'] {
+      @include iui-focus;
+      cursor: pointer;
+      -webkit-tap-highlight-color: transparent;
+      transition: background-color $iui-speed-fast ease-out;
+      @media (forced-colors: active) {
+        border: 1px solid transparent;
+        border-bottom-width: 0;
+      }
+
+      // #region resets for button
+      @include iui-reset;
+      font-family: inherit;
+      background-color: unset;
+      color: unset;
+      // #endregion
+
+      &:hover {
+        background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+      }
+    }
+  }
+
+  @at-root a#{&} {
+    text-decoration: none;
+  }
+}
+
+@mixin iui-header-brand-icon {
+  width: $iui-icons-large;
+  height: $iui-icons-large;
+  margin: 0 $iui-side-navigation-icon-margins;
+  fill: var(--iui-icons-color-actionable);
+  will-change: transform;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed-fast ease-out;
+  }
+  @at-root .iui-slim & {
+    transform: scale(0.66666);
```

Comment:
I think this can be better handled through CSS variables so that you don't need to reference `iui-slim` inside the mixin. The css variable can be called something like `--_iui-header-brand-icon-scale` and the value can be changed inside `@mixin iui-page-header-slim`

Reply:
Great suggestion!  I need to reset my thinking, I'm used to not being able to use CSS variables like this.